src/helpers/jsonValidation.ts

```diff
@@ -0,0 +1,63 @@
+import _isArray from 'lodash/isArray';
+import _isObjectLike from 'lodash/isObjectLike';
+import { ConstructionSite } from '../types';
```

Comment:
should be auto linted with a line break with our new eslint/prettier config, right?

Reply:
the eslint import plugin is not merged yet due to issues with code climate.

i fixed it in 026308b2533032adc0542d2e374c7c51e81a27b4

Reply:
sure, but we want proper grouped imports, even if CodeClimate has not answered with a solution yet